Workflow/Sources/AnyWorkflowConvertible.swift

```diff
@@ -147,3 +147,18 @@ struct OutputBlockingWorkflow<Child: AnyWorkflowConvertible>: Workflow {
             .rendered(in: context)
     }
 }
+
+// MARK: -
+
+extension AnyWorkflowConvertible where Output == Never {
+    /// A convenience for workflows that don't produce output but want to change the declared output type.
+    ///
+    /// This equivalent to `.mapOutput { switch $0 {} }` but avoids the compiler warning for unreachable code in the provided transform.
```

Comment:
i think it'd be okay to drop this from the public declaration docs. technically i think you'd need a larger expression to even tell how the type conversion is happening. seems like it would be a good thing to keep as an internal comment though.

Reply:
Removed